File: tests/human/x/scala/match_full.scala
object match_full {
  def classify(n: Int): String = n match {
    case 0 => "zero"
    case 1 => "one"
    case _ => "many"
  }

  def main(args: Array[String]): Unit = {
    val x = 2
    val label = x match {
      case 1 => "one"
      case 2 => "two"
      case 3 => "three"
      case _ => "unknown"
    }
    println(label)

    val day = "sun"
    val mood = day match {
      case "mon" => "tired"
      case "fri" => "excited"
      case "sun" => "relaxed"
      case _ => "normal"
    }
    println(mood)

    val ok = true
    val status = if(ok) "confirmed" else "denied"
    println(status)

    println(classify(0))
    println(classify(5))
  }
}

File: tests/human/x/swift/match_full.swift
let x = 2
let label: String
switch x {
case 1: label = "one"
case 2: label = "two"
case 3: label = "three"
default: label = "unknown"
}
print(label)

let day = "sun"
let mood: String
switch day {
case "mon": mood = "tired"
case "fri": mood = "excited"
case "sun": mood = "relaxed"
default: mood = "normal"
}
print(mood)

let ok = true
let status = ok ? "confirmed" : "denied"
print(status)

func classify(_ n: Int) -> String {
    switch n {
    case 0: return "zero"
    case 1: return "one"
    default: return "many"
    }
}
print(classify(0))
